db/seeds.rb

```diff
@@ -5,24 +5,25 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
-User.delete_all
-Category.delete_all
-Test.delete_all
-Question.delete_all
-Answer.delete_all
+User.destroy_all
+Category.destroy_all
+Test.destroy_all
+Question.destroy_all
+Answer.destroy_all
+TestsUser.destroy_all
 
-User.create(name: 'Bill Murray')
-categories = Category.create([{ title: 'Ruby' }, { title: 'Ruby on Rails' }])
-tests = Test.create([{ title: 'Основы Руби',
-                       category_id: categories.first.id },
-                     { title: 'Rails-модели', level: 2,
-                       category_id: categories.last.id }])
-questions = Question.create([{ body: 'Как вычисляется квадратный корень',
-                               test_id: tests.first.id },
-                             { body: 'Какая команда генерирует модель?',
-                               test_id: tests.last.id }])
+user = User.create([{ name: 'Bill Murray' }, { name: 'Ghost Buster' }])
```

Comment:
тут в именах переменных наверное везде множественное число надо использовать
ты же по несколько объектов создаешь
```suggestion
users = User.create([{ name: 'Bill Murray' }, { name: 'Ghost Buster' }])
```